subprojects/test-runner/device-provider/impl/src/main/kotlin/com/avito/android/runner/devices/internal/kubernetes/KubernetesReservationFactory.kt

```diff
@@ -0,0 +1,48 @@
+package com.avito.android.runner.devices.internal.kubernetes
+
+import com.avito.android.runner.devices.internal.EmulatorsLogsReporter
+import com.avito.k8s.KubernetesApi
+import com.avito.logger.LoggerFactory
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.sync.Mutex
+
+@ExperimentalCoroutinesApi
+internal class KubernetesReservationFactory(
```

Comment:
`KubernetesReservationFactory` -> `KubernetesReservationClientFactory` ?

It creates a "client", but not a "reservation".